rdr_service/model/bq_base.py

```diff
@@ -567,6 +568,9 @@ def update(dest, src, schema):
             :return: dict
             """
             for key, val in src.items():
+                if not isinstance(key, str):
+                    logging.warning('Dict key is not a string.')
```

Comment:
Just in case this is triggered on Prod, should there be something in the log statement about what is being set and what it's being set on (to help track down what happened if it does get printed in Prod)?

Reply:
I don't expect this to hit in prod, I think it's just some low quality questionnaire response data in the lower environments causing this to hit. I don't think there will be a lot of hits in the lower environments.